src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -71,12 +74,24 @@ let nextContentId: number = 0;
 })
 export class SkyRepeaterItemComponent implements OnDestroy, OnInit, AfterViewInit {
 
+  /**
+   * Specifies configuration options for the buttons to display on an inline form
+   * within the repeater. This property accepts a `SkyInlineFormConfig` object.
```

Comment:
Based on our styles in phase 2 docs, we should remove the `This property accepts...` sentence. ... That said, the existing docs have a link to a section in the inline form docs where we describe the `SkyInineFormConfig` properties. Do we need such a link? Or to update the description here to briefly explain those properties?

Reply:
Oops...that was my mistake. I updated the text to provide a link. Good catch.